### PR TITLE
Fix repeat personnel roles bug

### PR DESCRIPTION
**Pull request by chiaberry**
## Associated issues

https://github.com/cityofaustin/atd-data-tech/issues/8025

## Testing
**URL to test:** 
database changes, so test on https://moped-test.austinmobility.io/moped/

**Steps to test:**

On staging/production right now, if you have team members associated with a project as well as more than one 
project partner, you will find that the personnel roles are repeated. 
Screen Shot 2022-01-05 at 9 47 07 AM

To test, add more than one team member to a project as well as more than one project partner. Confirm nothing is repeated. 

---
#### Ship list
- [ ] Code reviewed 
- [x] Product manager approved
- [ ] ~Added to [QA test script](https://docs.google.com/spreadsheets/d/1n_O6MLh9cwwPf57HUM394Ea-z9uuoEV1-QW4axNZXLE/edit#gid=776973707)~

### CODE REVIEW

File: moped-database/migrations/1641337581998_run_sql_migration/up.sql [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    mp.start_date,
+    mp.fiscal_year,
+    mp.capitally_funded,
+    mp.project_priority,
+    mp.date_added,
+    mp.added_by,
+    mp.is_retired,
+    mp.milestone_id,
+    COALESCE(mp.status_id, 0) AS status_id,
+    string_agg(distinct concat(mu.first_name, ' ', mu.last_name, ':', mpr.project_role_name), ','::text) AS project_team_members,
```
Comment (chiaberry) [inline moped-database/migrations/1641337581998_run_sql_migration/up.sql:24]: adding "distinct" here and on line 27 is the change

File: moped-editor/src/views/projects/projectsListView/ProjectsListViewFiltersConf.js [hunk 2]
```diff
@@ -66,7 +66,7 @@ export const ProjectsListViewFiltersConf = {
     },
     {
       name: "current_phase",
-      label: "Currrent phase",
+      label: "Current phase",
```
Comment (chiaberry) [inline moped-editor/src/views/projects/projectsListView/ProjectsListViewFiltersConf.js:69]: 😬

**Review by sergiogcx — APPROVED** [review]

**Review by amenity — APPROVED** [review]
Beautiful, thanks @chiaberry 🙏

**Review by frankhereford — APPROVED** [review]
Awesome - thanks for fixing that! 🚀 🌔 🌠

**Pull request merged**